### DISCUSSION TITLE
Caddy v2

### DISCUSSION THREAD

**Discussion by louis-l** [vitodeploy/vito#156] [Q&A]
Is there a way to use Caddy v2 instead of Nginx?

I have Laravel site already running via Caddy, Im concerned about the performance when transfering back to Nginx. 

Cheers

**Comment by saeedvaziry** [comment]
@louis-l it's not prio at the moment to have 2 web servers supported but vito has enough abstraction to support it in the future

**Reply by louis-l** [reply]
Awesome, looking forward to it.

**Comment by saeedvaziry** [comment]
We will continue with one web server for now
